server/sonar-web/src/main/js/apps/account/organizations/CreateOrganizationForm.js

```diff
@@ -229,7 +229,7 @@ class CreateOrganizationForm extends React.PureComponent {
           </div>
 
           <footer className="modal-foot">
-            {this.state.processing
+            {this.state.loading
```

Comment:
No, only the submit button. There is nothing really bad if you click "Cancel", because it just closes the modal, and not sends a request.